src/RadworksGovernor.sol

```diff
@@ -0,0 +1,232 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
+import {Governor} from "@openzeppelin/contracts/governance/Governor.sol";
+import {IERC165} from "@openzeppelin/contracts/interfaces/IERC165.sol";
+import {
+  ERC20VotesComp,
+  GovernorVotesComp
+} from "@openzeppelin/contracts/governance/extensions/GovernorVotesComp.sol";
+import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
+import {ICompoundTimelock} from
+  "@openzeppelin/contracts/governance/extensions/GovernorTimelockCompound.sol";
+import {GovernorSettings} from "@openzeppelin/contracts/governance/extensions/GovernorSettings.sol";
+
+import {IRadworksTimelock} from "src/interfaces/IRadworksTimelock.sol";
+import {GovernorTimelockCompound} from "src/lib/GovernorTimelockCompound.sol";
+import {GovernorCompatibilityBravo} from
+  "@openzeppelin/contracts/governance/compatibility/GovernorCompatibilityBravo.sol";
+
+/// @notice The upgraded Radworks Governor: Bravo compatible and built with OpenZeppelin.
```

Comment:
nit, feel free to ignore

```suggestion
/// @notice The upgraded Radworks Governor: Bravo compatible and extended from OpenZeppelin.
```

Reply:
Done 4c8b4aa5079ff2ca8a0124eeab3624cc5cfce0db